cocos/core/root.ts

```diff
@@ -342,7 +342,7 @@ export class Root {
      * @param width The new width of the window.
      * @param height The new height of the window.
      */
-    public resize (width: number, height: number) {
+    public resize (windowId: number, width: number, height: number) {
```

Comment:
Why put window id at first parameter. Does widow id exist on web platform?

Reply:
I'll change it to the last optional paremeter.